src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -2237,7 +2239,9 @@ GenTree* Compiler::convertHWIntrinsicToMask(var_types   type,
 //
 GenTree* Compiler::convertHWIntrinsicFromMask(GenTreeHWIntrinsic* node, var_types type)
```

Comment:
nit: I know this was introduced this way in the other PR, but the typical convention is `gtNew*Node` and to expose these types of APIs in `gentree.cpp`

This is a case where we have a similar API needed for all architectures supporting `TYP_MASK` and so we'll have less overall churn by putting it in the right place and giving it a consistent name.

I imagine that `gtNewSimdConvertVectorToMaskNode` and `gtNewSimdConvertMaskToVectorNode` are the best choices here and fit in well with the names used by each architecture (`NI_*_ConvertVectorToMask` and `NI_*_ConvertMaskToVector`, where `*` is `SVE` or `AVX512F`